test/SideNav.spec.js

```diff
@@ -26,71 +26,92 @@ function setup(props = {}, children, mounted) {
-    const { sideNav } = setup({ className: 'red', fixed: true });
-    expect(sideNav).toMatchSnapshot();
-  });
+    test('calls SideNav', () => {
+      mount(<SideNav />);
+      expect(sidenavInitMock).toHaveBeenCalledTimes(1);
+    });
 
-  test('should render fixed if not passed trigger or fixed', () => {
-    const { wrapper } = setup({ className: 'red' });
-    expect(wrapper).toMatchSnapshot();
-  });
+    test('renders', () => {
+      const { sideNav } = setup();
+      expect(sideNav).toMatchSnapshot();
+    });
 
-  test('should render trigger on large screens not fixed`', () => {
-    const { trigger } = setup({ className: 'green' });
-    expect(trigger).toMatchSnapshot();
-  });
+    test('can be `fixed`', () => {
+      const { sideNav } = setup({ className: 'red', fixed: true });
+      expect(sideNav).toMatchSnapshot();
+    });
 
-  test('should be fixed if no trigger is passed', () => {
-    const component = <SideNav className="red" />;
-    const wrapper = shallow(component);
-    const sideNav = wrapper.find('.sidenav');
-    expect(sideNav).toMatchSnapshot();
-  });
+    test('should render fixed if not passed trigger or fixed', () => {
+      const { wrapper } = setup({ className: 'red' });
+      expect(wrapper).toMatchSnapshot();
+    });
 
-  test('should render a given id', () => {
-    const { sideNavProps, triggerProps } = setup({ id: 'test' });
-    expect(sideNavProps.id).toEqual('test');
-    expect(triggerProps['data-target']).toEqual('test');
-  });
+    test('should render trigger on large screens not fixed`', () => {
+      const { trigger } = setup({ className: 'green' });
+      expect(trigger).toMatchSnapshot();
+    });
 
-  test('should render children', () => {
-    const { sideNav } = setup({}, <span className="test-child" />);
-    const children = sideNav.find('.test-child');
+    test('should be fixed if no trigger is passed', () => {
+      const component = <SideNav className="red" />;
+      const wrapper = shallow(component);
+      const sideNav = wrapper.find('.sidenav');
+      expect(sideNav).toMatchSnapshot();
+    });
 
-    expect(children).toHaveLength(1);
-  });
+    test('should render a given id', () => {
+      const { sideNavProps, triggerProps } = setup({ id: 'test' });
+      expect(sideNavProps.id).toEqual('test');
+      expect(triggerProps['data-target']).toEqual('test');
+    });
 
-  test('should consume props it uses', () => {
-    const { sideNavProps } = setup({
-      id: 'test123',
-      options: {},
-      shouldTransfer: true
+    test('should render children', () => {
+      const { sideNav } = setup({}, <span className="test-child" />);
+      const children = sideNav.find('.test-child');
+
+      expect(children).toHaveLength(1);
     });
 
-    expect(sideNavProps.trigger).toBeUndefined();
-    expect(sideNavProps.options).toBeUndefined();
-    expect(sideNavProps.shouldTransfer).toBeTruthy();
-  });
+    test('should consume props it uses', () => {
```

Comment:
I don't understand this spec, what do you mean by consume the props it uses?

Reply:
Not sure, I didn't write this test, it was already there